src/OpenTelemetry.AutoInstrumentation/Configurations/MetricSettings.cs

```diff
@@ -62,32 +62,54 @@ protected override void OnLoad(Configuration configuration)
         MetricsEnabled = configuration.GetBool(ConfigurationKeys.Metrics.MetricsEnabled) ?? true;
     }
 
-    private static MetricsExporter ParseMetricExporter(Configuration configuration)
+    private static IReadOnlyList<MetricsExporter> ParseMetricExporter(Configuration configuration)
     {
-        var metricsExporterEnvVar = configuration.GetString(ConfigurationKeys.Metrics.Exporter)
-                                    ?? Constants.ConfigurationValues.Exporters.Otlp;
+        var metricsExporterEnvVar = configuration.GetString(ConfigurationKeys.Metrics.Exporter);
 
-        switch (metricsExporterEnvVar)
+        if (string.IsNullOrWhiteSpace(metricsExporterEnvVar))
         {
-            case null:
-            case "":
-            case Constants.ConfigurationValues.Exporters.Otlp:
-                return MetricsExporter.Otlp;
-            case Constants.ConfigurationValues.Exporters.Prometheus:
-                return MetricsExporter.Prometheus;
-            case Constants.ConfigurationValues.None:
-                return MetricsExporter.None;
-            default:
-                if (configuration.FailFast)
-                {
-                    var message = "Metric exporter '{metricsExporterEnvVar}' is not supported.";
-                    Logger.Error(message);
-                    throw new NotSupportedException(message);
-                }
-
-                Logger.Error($"Metric exporter '{metricsExporterEnvVar}' is not supported. Defaulting to '{Constants.ConfigurationValues.Exporters.Otlp}'.");
-
-                return MetricsExporter.Otlp;
+            return new List<MetricsExporter> { MetricsExporter.Otlp }.AsReadOnly();
         }
+
+        var exporters = new HashSet<MetricsExporter>();
+        var exporterNames = metricsExporterEnvVar!.Split(',')
+                                                  .Select(e => e.Trim())
+                                                  .Where(e => !string.IsNullOrEmpty(e))
+                                                  .ToList();
+
+        exporters.Add(MetricsExporter.None);
```

Comment:
Related to https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/3523/files#r1687773496